CONTRIBUTING.md

```diff
@@ -1,4 +1,14 @@
-# Tox Setup instructions
+# Contributing to Tempo
+We happily welcome contributions to Tempo. 
+We use [GitHub Issues](https://github.com/databrickslabs/tempo/issues) to track community reported issues 
+and [GitHub Pull Requests](https://github.com/databrickslabs/tempo/pulls) for reviewing and accepting code changes.
+
+If you have found a bug or other issue with the library, please [open an new issue](https://github.com/databrickslabs/tempo/issues/new) to report it.
+If you have written a patch for a bug or new feature, please [submit a pull request](https://github.com/databrickslabs/tempo/compare) and we will be happy to review it for inclusion in a future release.
+
+You can [create a fork of the code](https://github.com/databrickslabs/tempo/fork) to begin your own modifications. 
+
+## Tox Setup instructions
```

Comment:
Shouldn't tox be in the README.md and documentation? I think pandas is a good example of placing build instructions in the official docs with BSD 3 license strictly having terms for contribution. Separately, building is up to the tempo project and should be in the github workflows (e.g. https://github.com/pandas-dev/pandas/blob/main/.github/workflows/wheels.yml)